net-p2p/syncthing/syncthing-0.14.12.ebuild

```diff
@@ -17,6 +17,8 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86 ~arm"
 IUSE="tools"
 
+DEPEND="tools? ( >=dev-lang/go-1.7.1:= )"
```

Comment:
@mgorny has recently gone on a complaint tour that `USE="tools"` is too generic to have any meaning. Could you be more specific about what `USE="tools"` actually does (maybe `USE="go"`)?